ceph/rados/cephvolume_workflows.py

```diff
@@ -0,0 +1,387 @@
+"""
+Test Module to perform specific functionalities of ceph-volume.
+ - ceph-volume lvm list [OSD_ID | DEVICE_PATH | VOLUME_GROUP/LOGICAL_VOLUME]
+ - ceph-volume lvm zap [--destroy] [--osd-id OSD_ID | --osd-fsid OSD_FSID | DEVICE_PATH]
+ - ceph-volume --help
+ - ceph-volume lvm --help
+ - ceph-volume lvm prepare --bluestore [--block.db --block.wal] --data path
+ - ceph-volume lvm activate [--bluestore OSD_ID OSD_FSID] [--all]
+ - ceph-volume lvm deactivate OSD_ID
+ - ceph-volume lvm create --bluestore --data VOLUME_GROUP/LOGICAL_VOLUME
+ - ceph-volume lvm batch --bluestore PATH_TO_DEVICE [PATH_TO_DEVICE]
+ - ceph-volume lvm migrate --osd-id OSD_ID --osd-fsid OSD_UUID
+    --from [ data | db | wal ] --target VOLUME_GROUP_NAME/LOGICAL_VOLUME_NAME
+ - ceph-volume lvm new-db --osd-id OSD_ID --osd-fsid OSD_FSID --target VOLUME_GROUP_NAME/LOGICAL_VOLUME_NAME
+ - ceph-volume lvm new-wal --osd-id OSD_ID --osd-fsid OSD_FSID --target VOLUME_GROUP_NAME/LOGICAL_VOLUME_NAME
+"""
+
+import json
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephVolumeWorkflows:
+    """
+    Contains various functions to verify ceph-volume commands
+    """
+
+    def __init__(self, node: CephAdmin):
+        """
+        initializes the env to run ceph-volume commands
+        Args:
+            node: CephAdmin object
+        """
+        self.rados_obj = RadosOrchestrator(node=node)
+        self.cluster = node.cluster
+        self.client = node.cluster.get_nodes(role="client")[0]
+
+    def run_cv_command(
+        self,
+        cmd: str,
+        host,
+        timeout: int = 300,
+    ) -> str:
+        """
+        Runs ceph-volume commands within cephadm shell
+        Args:
+            cmd: command that needs to be run
+            host: (ceph.utils.CephVMNode): CephVMNode
+            timeout: Maximum time allowed for execution.
+        Returns:
+            output of respective ceph-volume command in string format
+        """
+        _cmd = f"cephadm {cmd}"
+        try:
+            out, err = host.exec_command(sudo=True, cmd=_cmd, timeout=timeout)
+        except Exception as er:
+            log.error(f"Exception hit while command execution. {er}")
+            raise
+        return str(out)
+
+    def help(self, host):
+        """Module to run help command with ceph-volume to display usage
+         Args:
+            host: (ceph.utils.CephVMNode): CephVMNode
+        Returns:
+            Output of ceph-volume usage
+        """
+        _cmd = "ceph-volume --help"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_help(self, host):
+        """Module to run help command with ceph-volume lvm to display usage
+         Args:
+            host: (ceph.utils.CephVMNode): CephVMNode
+        Returns:
+            Output of ceph-volume lvm usage
+        """
+        _cmd = "ceph-volume --help"
```

Comment:
command used here is same as in `def help()`. command update needed?

Reply:
Updated the command
https://github.com/red-hat-storage/cephci/blob/780718b450dcba550afcc5dc98cd81f5ef20d2fd/ceph/rados/cephvolume_workflows.py#L86C9-L86C40